src/nerve/server/proxy_manager.py

```
@@ -0,0 +1,423 @@
+        # Stop all proxies concurrently
+        await asyncio.gather(
+            *[self.stop_proxy(node_id) for node_id in list(self._proxies.keys())],
+            return_exceptions=True,
+        )
+
+    async def _create_passthrough_proxy(
+        self,
+        port: int,
+        config: ProviderConfig,
+        debug_dir: str | None,
+    ) -> Any:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `_create_passthrough_proxy`

(ANN401)